src/components/modal/__test__/index.spec.tsx

```diff
@@ -10,9 +10,20 @@ const TestCase = (props: Omit<ModalProps, 'testID'>) => {
 };
 
 describe('Sanity modal test', () => {
-  it('Should be visible', () => {
-    const renderTree = render(<TestCase visible/>);
-    const modal = ModalDriver({renderTree, testID});
-    expect(modal.isVisible()).toBeTruthy();
+  it('Should be not visible at first and visible at second render', () => {
+    const renderTree = render(<TestCase/>);
+    const modalDriver = ModalDriver({renderTree, testID});
+    expect(modalDriver.isVisible()).toBeFalsy();
+    renderTree.rerender(<TestCase visible/>);
+    expect(modalDriver.isVisible()).toBeTruthy();
+  });
+
+  it('Should press on background', () => {
```

Comment:
Test close when background is pressed

Reply:
Or Should dismiss modal on background press

Reply:
I actually don't think this causes the modal to dismiss it just lets you press on the background. On the dialog pressing on the background of the modal makes it dismiss.